test/integration_helper.rb

```diff
@@ -44,7 +44,14 @@ def run
 
     def installed?
       return @installed unless @installed.nil?
-      s = server == 'HTTP' ? 'net/http/server' : server
+      s = case server
+      when 'HTTP'
+        'net/http/server'
+      when 'puma'
+        'puma/rack/handler'
```

Comment:
Tests output show `puma is not installed, skipping integration tests` so I think this _require_ is not working as expected.

Reply:
Oh good catch 😬 Explains why things went so smooth 😆